Source/WebCore/layout/formattingContexts/inline/text/TextUtil.cpp

```diff
@@ -499,7 +500,7 @@ TextDirection TextUtil::directionForTextContent(StringView content)
 {
     if (content.is8Bit())
         return TextDirection::LTR;
-    return ubidi_getBaseDirection(content.characters16(), content.length()) == UBIDI_RTL ? TextDirection::RTL : TextDirection::LTR;
+    return ubidi_getBaseDirection(content.span16().data(), content.length()) == UBIDI_RTL ? TextDirection::RTL : TextDirection::LTR;
```

Comment:
Seems like the many places like this should use arguments like `content.span16().size()` instead of `content.length()`.